src/deploy-cromwell-on-azure/scripts/docker-compose.yml

```diff
@@ -44,6 +44,7 @@ services:
       - type: bind
         source: /cromwellazure/cromwell-tmp
         target: /cromwell-tmp
+      - /cromwellazure/cromwell-application.conf:/configuration/cromwell-application.conf:rw
```

Comment:
The list of volumes to mount is dynamically created at startup. This includes the "configuration" storage container.